components/combobox/src/auro-combobox.js

```diff
@@ -636,14 +621,16 @@ export class AuroCombobox extends LitElement {
    */
   configureInput() {
     // When input is in bibtemplate, make the event to be fired at combobox element
-    this.bubbleUpInputKeyEvent = this.bubbleUpInputKeyEvent.bind(this);
-    this.input.addEventListener('keydown', this.bubbleUpInputKeyEvent);
-    this.input.addEventListener('keyup', this.bubbleUpInputKeyEvent);
-    this.input.addEventListener('input', this.bubbleUpInputKeyEvent);
+    this.bubbleUpInputEvent = this.bubbleUpInputEvent.bind(this);
+    this.input.addEventListener('keydown', this.bubbleUpInputEvent);
+    this.input.addEventListener('keyup', this.bubbleUpInputEvent);
+    this.input.addEventListener('input', this.bubbleUpInputEvent);
```

Comment:
One last small refinement on this to make it easier to extend in the future:

```suggestion
const events = ['keydown', 'keyup', 'input'];

events.forEach(event => {
  this.input.addEventListener(event, (e) => {
    this.bubbleUpInputEvent);
  });
});
```